### ISSUE TITLE
TypeError: The header content contains invalid characters

### ISSUE THREAD

**Issue by biga816** [awslabs/aws-serverless-express#15]
I got a error below when I set Japanese to post parameter.
Is it a bug?

```
2016-11-01T15:46:46.806Z	653a43f4-a04a-11e6-8038-c5c1480ca076	TypeError: The header content contains invalid characters
at ClientRequest.OutgoingMessage.setHeader (_http_outgoing.js:351:13)
at new ClientRequest (_http_client.js:79:14)
at Object.exports.request (http.js:31:10)
at forwardRequestToNodeServer (/var/task/node_modules/aws-serverless-express/index.js:75:22)
at Object.response.setEncoding.on.on.req.on.exports.createServer.exports.proxy (/var/task/node_modules/aws-serverless-express/index.js:118:9)
at exports.handler (/var/task/lambda.js:6:60)
```

**Comment by brettstack** [comment]
It's possible this PR will fix your issue https://github.com/awslabs/aws-serverless-express/pull/14. Try using that fork. Let me know the outcome, and I'll look deeper if the problem persists.

**Comment by brettstack** [comment]
@biga816, I have just merged the PR and release 1.2.0. Please let me know if this resolves your issue.

**Comment by biga816** [comment]
It resolved my issue!!
Thank you so much:)
